hail/python/hail/experimental/sparse_split_multi.py

```diff
@@ -160,11 +152,23 @@ def with_pl(pl):
 
                     dropped_fields.append('LPL')
 
-                return hl.cond(hl.len(ds.alleles) == 1,
-                                   old_entry.annotate(**{f[1:]: old_entry[f] for f in ['LGT', 'LPGT', 'LAD', 'LPL'] if f in fields}).drop(*dropped_fields),
-                                   old_entry.annotate(**new_exprs).drop(*dropped_fields))
+                return (hl.case()
+                        .when(hl.len(ds.alleles) == 1,
+                              old_entry.annotate(**{f[1:]: old_entry[f] for f in ['LGT', 'LPGT', 'LAD', 'LPL'] if f in fields}).drop(*dropped_fields))
+                        .when(hl.or_else(old_entry.LGT.is_hom_ref(), False),
+                            old_entry.annotate(**{f[1:]: old_entry[f] if f in ['LGT', 'LPGT'] else new_exprs[f[1:]]
```

Comment:
Maybe outside the scope, but it feels like `LAD` and `LPL` are untouched also, no? Unless I'm misremembering how this works, shouldn't they just go from `[0]` -> `[0]` and `NA` -> `NA`?

Reply:
oh, right, never mind. I forgot that we were unlocalizing or whatever we're calling that step.